libs/packages/sam-formly/src/lib/formly/wrappers/group.wrapper.ts

```diff
@@ -7,36 +7,63 @@ import * as qs from 'qs';
  */
 @Component({
   template: `
-  <ng-container [ngSwitch]="to.group">
-    <ng-container *ngSwitchCase="'accordion'">
-      <sds-accordion multi="true" displayMode="basic">
-        <sds-accordion-item class="sds-accordion__panel" [expanded]="modelHasValue()">
-          <sds-accordion-item-header> <span *ngIf="!to.hideLabel" [attr.aria-hidden]="!to.announceLabel ? undefined : 'true'"> {{to.label}} </span> </sds-accordion-item-header>
-          <ng-container #fieldComponent></ng-container>
-        </sds-accordion-item>
-      </sds-accordion>
-    </ng-container>
-    <ng-container *ngSwitchCase="'panel'">
-      <div class="sds-panel" [ngClass]="{'sds-panel--multiple' : field?.fieldGroup?.length }">
-        <div class="sds-panel__header" *ngIf="!to.hideLabel" [attr.aria-hidden]="!to.announceLabel ? undefined : 'true'"> {{to.label}} </div>
-        <div class="sds-panel__body">
-          <ng-container #fieldComponent></ng-container>
+    <ng-container [ngSwitch]="to.group">
```

Comment:
something causing templates to go over panel... not happening on master
<img width="1167" alt="Screen Shot 2020-05-20 at 5 04 30 PM" src="https://user-images.githubusercontent.com/63304927/82509216-0fede700-9abc-11ea-8b9c-fac954c1d735.png">

Reply:
Don't see any issues
![Screen Shot 2020-05-21 at 1 49 21 PM](https://user-images.githubusercontent.com/44271677/82589358-04f19000-9b6a-11ea-9e61-7cb7d5fbdfc7.png)